opentelemetry-sdk/CHANGELOG.md

```diff
@@ -28,6 +28,10 @@
   - After `shutdown`, `LogProcessor` will not process any new logs
 - Moving LogRecord implementation to the SDK. [1702](https://github.com/open-telemetry/opentelemetry-rust/pull/1702).
     - Relocated `LogRecord` struct to SDK, as an implementation for the trait in the API.
+- Clean up public methods in SDK.
```

Comment:
lets split this PR into smaller, focused PRs.

Reply:
Specifically, the jaeger cleanup can be a separate one.

Reply:
> jaeger cleanup

No longer needed